.clang-format

```diff
@@ -0,0 +1,5 @@
+# Complete list of style options can be found at: 
+# http://clang.llvm.org/docs/ClangFormatStyleOptions.html
+---
+BasedOnStyle: Google
```

Comment:
It was once mentioned.
We've borrowed all format facilities from facebook/rocksdb, but forgot to copy the `.clang-format` configuration file. This is the fix.

ref: https://github.com/facebook/rocksdb/blob/master/.clang-format